main.py

```diff
@@ -1,10 +1,51 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as data:
+        player_info = json.load(data)
+
+    for player, info in player_info.items():
+        race_name = info["race"]["name"]
+        skills = info["race"]["skills"]
+        guild = info["guild"]["name"] if info["guild"] else None
+
+        if not Race.objects.filter(name=race_name).exists():
+            race_description = info["race"]["description"]
+            Race.objects.create(name=race_name, description=race_description)
+
+        race = Race.objects.get(name=race_name)
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create

Reply:
The same in other places